library/open-liberty

```diff
@@ -16,7 +16,7 @@ Architectures: amd64, ppc64le, s390x
 File: Dockerfile.ubuntu.adoptopenjdk11
 
 Tags: kernel-slim, kernel-slim-java8-openj9
-Directory: releases/latest/kernel-slim
+Directory: releases/latest/kernel-slim21.0.0.8-release
```

Comment:
```console
failed "git show" for "releases/latest/kernel-slim21.0.0.8-release/Dockerfile.ubuntu.adoptopenjdk8" from commit "3314e0761eab88e38d67af9aa5d678eadec09043"
```

:eyes: GitHub action is unable to find a `Dockerfile.ubuntu.adoptopenjdk8` in the `releases/latest/kernel-slim21.0.0.8-release` directory. Perhaps this line was changed by accident.

Reply:
Yes, this is some scripting being too aggressive. I'll clean it up!